dali/imgcodec/decoders/nvjpeg/nvjpeg.cc

```diff
@@ -66,15 +66,15 @@ NvJpegDecoderInstance(int device_id, const std::map<std::string, any> &params)
 
   if (host_memory_padding_ > 0) {
     CUDA_CALL(nvjpegSetPinnedMemoryPadding(host_memory_padding_, nvjpeg_handle_));
-    for (auto thread_id : tp_->GetThreadIds()) {
+    for (const auto& thread_id : tp_->GetThreadIds()) {
```

Comment:
I thought that `thread_id` is rather a lightweight object (a number).

Reply:
You are right. It was marked as fixes submitted but the fix was not submitted, that's why I did it. Let me revert